src/__stories__/Forms/Input.tsx

```diff
@@ -22,18 +22,22 @@ const CustomInput = forwardRef<HTMLInputElement, IInput>(function CustomInput({
   return <Input {...props} ref={nodeRef} />;
 });
 
-export const inputKnobs = (tabName = 'Input'): IInput => ({
-  ...omit(boxKnobs(), 'opacity'),
-  disabled: boolean('disabled', false, tabName),
-  type: select('type', InlineInputType, 'text', tabName),
-  placeholder: text('placeholder', 'placeholder', tabName),
-  invalid: boolean('invalid', false, tabName),
-});
-
-export const autosizeInputKnobs = (tabName = 'Input'): IInput => ({
-  ...inputKnobs(),
-  type: select('type', AutosizeInputType, 'text', tabName),
-});
+export const inputKnobs = (tabName = 'Input'): IInput => {
+  return cleanKnobs({
+    ...omit(boxKnobs(), 'opacity'),
```

Comment:
Why do we omit `opacity`?

Reply:
@P0lip honestly I have no idea? It was there before, but will remove